### PR TITLE
[network] correct item types in sitemap

### PR DESCRIPTION
**Pull request by mueller-ma**
I cannot switch the device, so Switch doesn't make sense here

### CODE REVIEW

File: addons/binding/org.openhab.binding.network/README.md [hunk 1]
```diff
@@ -104,8 +104,8 @@ demo.sitemap:
 sitemap demo label="Main Menu"
 {
 	Frame {
-		Switch item=MyDevice
-		Number item=MyDeviceResponseTime
+		Text item=MyDevice
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.network/README.md]: Did you test that that text widgets really show the current state and not just the item name?

**Comment by ThomDietrich** [comment]
* `Switch item=MyDevice` shows the device correctly online and offline as a switch
* `Text item=MyDevice label="Device [%s]"` will show the device state (ON or OFF) as a text
* `Text item=MyDevice label="Device [MAP(active.map):%s]"` would depend on a matching map file and show the device state as e.g. "ein" and "aus"

All three and possibly more combinations are correct. The first one as in the docs is the most basic, the third is probably the one you want to use in your own sitemap.

Now "I cannot switch the device, so Switch doesn't make sense here" is a completely different story. To this day openHAB doesn't have a read-only binary state indicator. Many simply use a switch in this case. A disabled switch was under discussion at some point...

**Comment by mueller-ma** [comment]
@ThomDietrich I didn't want to add a mapping, because there are only mappings for de/en closed/open by default in openhab

File: addons/binding/org.openhab.binding.network/README.md (hunk 1 above)
Comment (mueller-ma) [inline addons/binding/org.openhab.binding.network/README.md]: @kaikreuzer I forget. Added label now

**Comment by watou** [comment]
> To this day openHAB doesn't have a read-only binary state indicator. Many simply use a switch in this case. A disabled switch was under discussion at some point...

Well, Contact items do not receive commands and appear "read only" in the UIs (you can't command a reed switch to open a door, for example).  But Switch items aren't a terrible choice for binary states if they are presented without UI control to send them commands that make no sense!

**Comment by ThomDietrich** [comment]
Switch items are not a terrible choice, they are just not quite right. A real-world dashboard would use an LED or some other not-switchable indicator. Welcome once again to the hell which is called usability :smile: 

Back to the topic: @mueller-ma your latest change **l**ooks **g**ood **t**o **m**e.

**Pull request merged**